.travis.yml

```diff
@@ -1,18 +1,23 @@
 language: ruby
 sudo: false
 cache: bundler
-script: 'bundle exec rake test:coverage --trace'
+script: 'bundle exec rubocop && bundle exec rake test:coverage --trace'
+after_script: 'echo `env`'
 rvm:
-  - 2.2.5
   - 2.3.1
   - jruby-9.0.5.0
   - ruby-head
   - jruby-head
+env:
+  - DB=sqlite
+  - DB=postgresql
+  - DB=mysql
 
 addons:
   postgresql: '9.4'
```

Comment:
FYI, we could add 9.5, if we want to use Travis's `trusty` build (which is is beta). `precise` is the default and doesn't support 9.5 yet though
https://docs.travis-ci.com/user/database-setup/#PostgreSQL